monte_carlo/integrator.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+
+class Integrator:
+    def __init__(self, 
+                pair_energy_object, 
+                cutoff = 3.0, 
+                low_acceptance = 0.38, 
+                high_acceptance = 0.42):
+        self.pair_energy_object = pair_energy_object
+        self.cutoff = cutoff
+        self.low_acceptance = low_acceptance
+        self.high_acceptance = high_acceptance
+
+    def __str__(self):
+        return 'Intergrator:' 
+
+## tocheck /w box_object & self.pair_energy_object
+    def get_particle_energy(self, 
+                            coordinates, 
+                            i_particle, 
+                            box_object):
+        ''' Computes the energy of a particle with the rest of the system.
+
+        ----------
+        Parameters
+        ----------
+        coordinates : np.array
+            An array of atomic coordinates (x, y, z). Shape (n, 3), where
+            n is the number of particles.
+        box_length : float
+            The dimensions of the square box in reduced units.
+        i_particle : np.array
```

Comment:
I think you mean the i-th particle within the coordinates. This may not need to be a parameter. You can write the ith-particle by range(len(coordinates))